### DISCUSSION TITLE
2 day event shows only enddate

### DISCUSSION THREAD

**Discussion by ghost** [add2cal/add-to-calendar-button#463] [Q&A] [answered]
I have a 2 day event that starts on oct 19, and ends on oct 20 these are the settings.

When downloading the event (Apple) it shows only 20 oct

![image](https://github.com/add2cal/add-to-calendar-button/assets/1188482/aa92245c-fa22-40b5-9b76-c82b2cdf4981)

you can check it live here: https://libertas-music.com/amsterdam-dance-event-2023/

Included the generated ics
[event-to-save-in-my-calendar.ics.zip](https://github.com/add2cal/add-to-calendar-button/files/12598811/event-to-save-in-my-calendar.ics.zip)

Maybe I'm missing something?

Cheers,

Cyriel

**Comment by jekuer** [comment, marked as answer]
I assume that your Timezone is GMT+2?
The button defaults to GMT. Therefore, when you generate the event, from your location it is 2h off.
Add the timeZone option to fix it.
like timeZone="Europe/Amsterdam"

**Reply by ghost** [reply]
Yes, you were correct. Timezone Fixed it.
